### PR TITLE
Improve configurations

### PR DESCRIPTION
**Pull request by waldyr**
Taken from `mg_v2`.

### CODE REVIEW

**Comment by gerrywastaken** [comment]
These are some fantastic changes. Simplifying code, make things easier to read and put the version in a single spot. Very happy.

@maurogeorge Thank you for submitting them
@waldyr Thank you for spotting them and extracting them out

**Comment by waldyr** [comment]
@gerrywastaken @michaeldv Any objections to this?

**Comment by michaeldv** [comment]
LGTM

**Comment by gerrywastaken** [comment]
@waldyr just in case my 👍  on your post and previous comment about these being fantastic changes wasn't clear...

LGTM :)

Othe than the merge conflicts which now seemed to have creeped in.

File: lib/awesome_print/custom_defaults.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          object.ai
+        end
+      end.new
+    end
+
+    def usual_rb
+      IRB::Irb.class_eval do
+        def output_value
+          ap @context.last_value
+        end
```
Comment (gerrywastaken) [inline lib/awesome_print/custom_defaults.rb]: I think it's just missing

```ruby
rescue NoMethodError        
  puts "(Object doesn't support #ai)"
```

Which was recently added by a pull request.

#212

File: lib/awesome_print/inspector.rb [hunk 2]
```diff
@@ -6,48 +6,6 @@
...
-
-    def irb!
-      return unless defined?(IRB)
-      unless IRB.version.include?('DietRB')
-        IRB::Irb.class_eval do
-          def output_value
-            ap @context.last_value
-          rescue NoMethodError
-            puts "(Object doesn't support #ai)"
-          end
```
Comment (gerrywastaken) [inline lib/awesome_print/inspector.rb]: ^ The missing bit in the new code.

**Comment by gerrywastaken** [comment]
@waldyr cool, looks ready to merge.

**Pull request merged**